### PR TITLE
Issue #1759 by Inlead: Christmas Calendar.

### PR DESCRIPTION
**Pull request by inlead**
http://platform.dandigbib.org/issues/1759

### CODE REVIEW

File: modules/ding_christmas_calendar/config.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# First, require any additional compass plugins installed on your system.
```
Comment (inlead) [inline modules/ding_christmas_calendar/config.rb]: File should be removed.

File: modules/ding_christmas_calendar/css/ding_christmas_calendar.css [hunk 2]
```diff
@@ -0,0 +1,269 @@
+/**
```
Comment (inlead) [inline modules/ding_christmas_calendar/css/ding_christmas_calendar.css]: Should be moved to `themes/ddbasic`.

File: modules/ding_christmas_calendar/config.rb (hunk 1 above)
Comment (guddo) [inline modules/ding_christmas_calendar/config.rb]: File should be removed.

File: modules/ding_christmas_calendar/css/ding_christmas_calendar.css (hunk 2 above)
Comment (guddo) [inline modules/ding_christmas_calendar/css/ding_christmas_calendar.css]: Should be moved to `themes/ddbasic`.

File: modules/ding_christmas_calendar/ding_christmas_calendar.info [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name = Ding Christmas calendar
+description = Represents christmas calendar widget based on quiz module.
+core = 7.x
+package = Ding!
+project = ding_christmas_calendar
+dependencies[] = ctools
+dependencies[] = multichoice
+dependencies[] = page_manager
+dependencies[] = quiz
```
Comment (guddo) [inline modules/ding_christmas_calendar/ding_christmas_calendar.info]: Modules should be added to `ding2.make`.

File: modules/ding_christmas_calendar/ding_christmas_calendar.install [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+/**
+ * Implements hook_uninstall().
+ */
+function ding_christmas_calendar_uninstall() {
+  field_delete_field('field_background');
+
+  $id = variable_get('ding_christmas_calendar_quiz_id', NULL);
+  if (!empty($id)) {
+//    $questions = ding_christmas_calendar_questions($id);
```
Comment (guddo) [inline modules/ding_christmas_calendar/ding_christmas_calendar.install]: To be removed?

File: modules/ding_christmas_calendar/sass/_mixins.scss [hunk 5]
```diff
@@ -0,0 +1,229 @@
+/**
```
Comment (guddo) [inline modules/ding_christmas_calendar/sass/_mixins.scss]: How does this differ from `themes/ddbasic`?

File: modules/ding_christmas_calendar/sass/ding_christmas_calendar.scss [hunk 6]
```diff
@@ -0,0 +1,292 @@
+@import "_mixins";
```
Comment (guddo) [inline modules/ding_christmas_calendar/sass/ding_christmas_calendar.scss]: Should be moved to `themes/ddbasic`.

**Comment by guddo** [comment]
- Theme-related things should be moved to `theme/ddbasic`.
- Translations file missing.
- Node export of all Quiz questions and answers missing.

**Pull request closed without merging**